### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by jotak**
- Using existing images built by CI
- note on pull policy
- applying CR
- configuriing OVS

### CODE REVIEW

**Review by jpinsonneau — APPROVED** [review]
LGTM

Should we mention usage of kustomize here or in general documentation ?

**Comment by jotak** [comment]
> Should we mention usage of kustomize here or in general documentation ?

Like what? Changing the target namespace, and so on? (I don't know, I never change these values, but if it's useful for you it probably can be useful also for someone else)

**Review by OlivierCazade — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,17 +8,55 @@ This operator uses the [Go Operator Framework](https://sdk.operatorframework.io/
...
 
-By default, the operator is deployed in namespace "network-observability".
+### Enabling OVS IPFIX export
+
+This part will eventually be done automatically by the operator, but for the time being it requires manual intervention.
+
+#### On KIND
+
+```bash
+GF_IP=`kubectl get svc goflow-kube -ojsonpath='{.spec.clusterIP}'` && echo $GF_IP
```
Comment (jpinsonneau) [inline README.md:48]: you need to add -n network-observability here

File: README.md [hunk 2]
```diff
@@ -8,17 +8,55 @@ This operator uses the [Go Operator Framework](https://sdk.operatorframework.io/
...
+kubectl set env daemonset/ovnkube-node -c ovnkube-node -n ovn-kubernetes OVN_IPFIX_TARGETS="$GF_IP:2055"
+```
+
+#### On OpenShift
+
+In OpenShift, a difference with the upstream `ovn-kubernetes` is that the flows export config is managed by the `ClusterNetworkOperator`.
+
+```bash
+GF_IP=`oc get svc goflow-kube -ojsonpath='{.spec.clusterIP}'` && echo $GF_IP
```
Comment (jpinsonneau) [inline README.md:57]: same here I guess ?

**Review by jpinsonneau — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -8,17 +8,55 @@ This operator uses the [Go Operator Framework](https://sdk.operatorframework.io/
...
+```
+
+Note, the default image pull policy is `IfNotPresent`, so if you previously deployed the operator on a cluster and then create another build with the same image name/tag, it won't be pulled in the cluster registry. So you need either to provide a different image name/tag for every build, or modify [manager.yaml](./config/manager/manager.yaml) to set `imagePullPolicy: Always`, then re-deploy.
+
+Then, you can deploy a custom resource, e.g.:
 
-# With custom image
-IMG="docker.io/myuser/netobserv:latest" make image-build image-push deploy
+```bash
+kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector.yaml
```
Comment (memodi) [inline README.md:38]: need to apply namespace here "-n network-observability"

Comment (jotak) [reply README.md:38]: actually, here there's no need to provide a namespace because this resource is cluster-scope, not namespace-scope

**Review by memodi — APPROVED** [review]

**Review by mariomac — APPROVED** [review]

**Pull request merged**